docs/contributing/code_contribution_guide.rst

```diff
@@ -90,6 +90,19 @@ branch which is your feature branch pushed at your fork.
 .. note::
     Do not push to ``master`` or ``devel`` branch on your fork or upstream.
 
+Best Practices when creating Pull Request
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- It's good practice to link PR to an issue during commit message. Such as stating ``Fix #132`` for fixing issue 132.
+
+- Please create a meaningful title and PR description to help outline your proposed changes.
+
+- Assign PR to yourself when creating the issue. You should @ mention (`@shahzebsiddiqui <https://github.com/shahzebsiddiqui>`_) the project maintainers to get their attention.
+
+- If your PR is not ready for review, please add ``WIP:`` to your PR title to indicate it's a work in progress and make it a draft PR. This will prevent maintainers from reviewing your PR until it's ready.
```

Comment:
Line 102: please  add "that" after "to indicate"

Reply:
fixed